_posts/2021-01-13-an-introduction-to-mixing-audio.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title:  "Introduction to Mixing Audio - How to not sound like a Dial Up Modem"
+author: flavius
+categories: [ audio ]
+image: "mixer.webp"
+featured: false
+---
+What is audio mixing? Is it helpful or a hindrance? What are my options for mixing audio?
+How much does it cost?
```

Comment:
Add to previous line?

Reply:
What is audio mixing? Is it helpful or a hindrance? What are my options for mixing audio and how much does it cost?

Reply:
Changed to the above